src/sections/Community/Handbook/repository.js

```diff
@@ -204,6 +209,64 @@ const Repository = () => {
                 </div>
               );
             })}
+            <a id="Maintainers">
+              <h2>Maintainers</h2>
+            </a>
+
+            {Maintainers.map((Maintainers, index) => {
+              const { category } = Maintainers;
+              return (
+                <div className="table-container" key={index}>
+                  <table className="frontendTable" key={category}>
+                    <thead>
+                      <tr>
+                        <th className="linkscol">Site</th>
+                        <th>Project</th>
+                        <th>Maintainers</th>
+                        <th className="linkscol">Repo</th>
+                      </tr>
+                    </thead>
+                    {Maintainers.subdata.map((subdata) => {
+                      const {
+                        project,
+                        maintainers_name,
+                        repository,
+                        site,
+                        image,
+                        link
+                      } = subdata;
+                      return (
+                        <tbody key={project}>
+                          <tr>
+                            <td>
+                              <a href={site} target="_blank" rel="noreferrer">
+                                <img className="site-icon" src={image} />
+                              </a>
+                            </td>
+                            <td>{project}</td>
+                            <td>
+                              <a href={link[0]}>{maintainers_name[0]}</a>
+                              <a href={link[1]}>{maintainers_name[1]}</a>
+                              <a href={link[2]}>{maintainers_name[2]}</a>
+                              <a href={link[3]}>{maintainers_name[3]}</a>
+                            </td>
```

Comment:
Use `map` here

Reply:
using map here results in a React error as shown in above ss too 
see link
https://github.com/layer5io/layer5/pull/3071#issuecomment-1214067467

Which looks completely valid to me , I also tried for loops but eventually ended up using this and this works pretty well.
For <5 number of  maintainers and I think this should work well as we will not have much no. of maintainers over time.

Reply:
Well, the code you wrote gave an error because of using the same variable name for the "iterator" and the list. Try updating it to something like `mname` or if you want `maintainer_name`. It will work like a charm :)

Reply:
> For <5 number of maintainers and I think this should work well as we will not have much no. of maintainers over time.

This is vague, and we shouldn't be writing code with such assumptions.

Reply:
Also, wrap the `return` statement within brackets like 
```
return (
   <a href={link[index]} rel="noreferrer" target="_blank" key={index}>{mname}</a>
);
```

Reply:
This should get you going along with he fix for `comma's`.
```
<td>
    {maintainers_name.map((mname, index) => {
      return (
        <a href={link[index]} rel="noreferrer" target="_blank" key={index}>
          <span>{index > 0 ? ", " : ""}</span>{mname}
        </a>
      );
    })}
</td>
```

Reply:
> Well, the code you wrote gave an error because of using the same variable name for the "iterator" and the list. Try updating it to something like `mname` or if you want `maintainer_name`. It will work like a charm :)

Woah, Now I know why it was giving error, I spent an hour trying to debug why it gave that error . 
Thanks I was also trying to use terniary operator for commas :+1:

Reply:
yes @Nikhil-Ladha everything works like a charm now that map error was kinda strange :sweat_smile: